### ISSUE TITLE
Read-only CSS is applied in autosuggest fields in submission wizard

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#8591]
**Describe the bug**
There are a few styles for form fields with the `readonly` attribute. These are being applied to elements (like a `<div>`) when not desired. Here is a screenshot of the autosuggest metadata fields in the submission wizard. The grey is a wrapper element:

![metadata](https://user-images.githubusercontent.com/2306629/215829514-160f0140-120a-486a-aae4-9b9895b2117e.png)

I found a comment in the code that seems to relate to this:

```css
// :read-only causes unexpected effects on editable fields
// that aren't input or textarea
input.pkpFormField__input:read-only,
textarea.pkpFormField__input:read-only {
	&:extend(.pkpFormField__input:disabled);
}
```

And it's fixed if I remove this general selector:

```diff
diff --git a/src/components/Form/fields/FieldBase.vue b/src/components/Form/fields/FieldBase.vue
index ca482b5..fbf8f97 100644
--- a/src/components/Form/fields/FieldBase.vue
+++ b/src/components/Form/fields/FieldBase.vue
@@ -281,8 +281,7 @@ export default {
                outline: 0;
        }
 
-       &:disabled,
-       &:read-only {
+       &:disabled {
                background: @bg-light;
                cursor: not-allowed;
                // Lighter than @text but accessible on @bg-light
```

@ewhanson I have a vague memory of this coming up before. Does this look like an ok change to make? I think the `input`/`textarea` selector was the fix, and maybe we don't need the general `&:read-only` rule?

**Comment by ewhanson** [comment]
@NateWr, yes, from the original issue I was running into the `input`/`textarea` selector fixed it. I tested the diff above and everything still looked good to me.

**Comment by NateWr** [comment]
Thanks! Fixed in https://github.com/pkp/ui-library/commit/83292b3b782df45c9fbc79399cc9289ebd64c1a2.
